webpack-helpers.js

```diff
@@ -0,0 +1,63 @@
+const path = require('path');
+
+function resolvePath(p) {
+    return path.resolve(__dirname, p);
+}
+
+function createThemeConfig(theme, themePath) {
+    return {
+        mode: 'production',
+        entry: {
+            index: [themePath],
+        },
+        output: {
+            path: resolvePath(`.build/uv-assets/themes/${theme}`),
+            publicPath: `/uv-assets/themes/${theme}`,
+        },
+        resolve: {
+            extensions: [".less", ".css"],
+        },
+        module: {
+            rules: [
+                {
+                    test: /\.css$/i,
+                    use: ["style-loader", "css-loader"],
+                },
+                {
+                    test: /\.less$/,
+                    use: [
+                        {
+                            loader: "css-loader",
+                        },
+                        {
+                            loader: "less-loader",
+                            options: {
+                                lessOptions: {
+                                    strictMath: true,
+                                },
+                                additionalData: '@theme: "uv-en-gb-theme";', // This should be unused.
```

Comment:
just wondering what this is for?

Reply:
This was a fix for less files that used that for importing paths. I think I caught all of them and could now be removed.